front/src/page/AddAudit.jsx

```diff
@@ -145,6 +168,238 @@ export const AddAudit = () => {
           </div>
         </div>
 
+        {/* backup scorecard section*/}
+        <div class="grid gap-4 mb-4 sm:grid-cols-2 sm:gap-6 sm:mb-5">
+          <div className="block w-[70%]">
+            <div class="flex items-center justify-between mb-5 gap-5">
+              <label
+                for="name"
+                class="block text-sm font-medium text-gray-900 dark:text-white"
+              >
+                Online On Premise Servers
+              </label>
+              <select
+                required
+                id="online-premise-servers-status"
+                name="online-premise-servers-status"
+                className=" w-[30%] bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-600 focus:border-primary-600 block p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
+              >
+                <option value="Protected">Protected</option>
+              </select>
```

Comment:
We can create and use our own select to remove code duplication


const CustomSelect = ({children, id, name}) => {
  return <select
  required
  id={id}
  name={name}
  className="mb-5 w-[50%] bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-600 focus:border-primary-600 block p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
>{children} </select>}


Use example:
`   <CustomSelect
                id="overall-information"
                name="overall-information"
              >
                <option value="Bronze">Bronze</option>
                <option value="Silver">Silver</option>
                <option value="Gold">Gold</option>
                <option value="Platinum">Platinum</option>
                <option value="No contract">No contract</option>
              </CustomSelect>`

Reply:
done